build/yamls/collector/Chart.yaml

```diff
@@ -0,0 +1,3 @@
+apiVersion: v2
+name: ipfix-collector
+version: 0.5.0
```

Comment:
I have added something to update the version number when we have a new release. It will be updated to the most updated release tag. Is that what you are suggesting?

Reply:
My understanding is that this is a template and we can use a generic value like <x.x.x>.

Btw, why does the name has to start with capital--"Chart.yaml"?
In addition, I see that Chart.yaml is not at all used when the mode is dev during the time of generation. Is this expected?

Reply:
Based on my understanding of the documentation https://helm.sh/docs/topics/charts/, under the "chart and versioning" subtitle, the overall structured files is called a chart, and the version implies the version of the chart. It is used when we apply command like `helm package`, which package a chart directory into a chart archive. So the value is decided by the contents of the current project. There are words like "_Helm v2 and later uses version numbers as release markers. Packages in repositories are identified by name plus version._". I'm not so sure on how should we assign the value of version, but I feel like the value is meaningful.

The naming and existence of Chart.yaml is also part of the Helm chart structure in order to make it configurable. https://helm.sh/docs/chart_template_guide/getting_started/
And it is case sensitive https://helm.sh/docs/chart_best_practices/conventions/#usage-of-the-words-helm-and-chart

Reply:
Sure. Two questions: 
1. I'm assuming you are saying we will edit the file `VERSION` and `Chart.yaml` in the generate-manifest script. From where shall we get the version value(`0.6.0`) to put in `VERSION`? Just do release version +1? Like 0.5.0 -> 0.6.0?
2. When shall we update the version value? Every time we have a new release we also upgrade the dev version by 0.1.0?

Reply:
No, I meant you should edit the Chart.yaml with the version from https://github.com/vmware/go-ipfix/blob/main/VERSION when you are generating the collector manifest using helm tool. Isn't that possible?

Reply:
That's definitely possible. I'll do that. I thought you want the VERSION file also to be edited in the generate-manifest script.